### ISSUE TITLE
Invalid version: "cd"

### ISSUE THREAD

**Issue by dogukanayd** [npm/npm#15817]
**This is my error:**
```
Dogukan-MacBook-Pro:~ Dogukan$ npm start
npm ERR! Darwin 16.4.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "start"
npm ERR! node v6.9.5
npm ERR! npm  v3.10.10

npm ERR! Invalid version: "cd"
npm ERR!
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/Dogukan/npm-debug.log
```

**This is my npm-debug.log**
```
0 info it worked if it ends with ok
1 verbose cli [ '/usr/local/bin/node', '/usr/local/bin/npm', 'start' ]
2 info using npm@3.10.10
3 info using node@v6.9.5
4 verbose config Skipping project config: /Users/Dogukan/.npmrc. (matches userconfig)
5 verbose stack Error: Invalid version: "cd"
5 verbose stack     at Object.fixVersionField (/usr/local/lib/node_modules/npm/node_modules/normalize-package-data/lib/fixer.js:191:13)
5 verbose stack     at /usr/local/lib/node_modules/npm/node_modules/normalize-package-data/lib/normalize.js:32:38
5 verbose stack     at Array.forEach (native)
5 verbose stack     at normalize (/usr/local/lib/node_modules/npm/node_modules/normalize-package-data/lib/normalize.js:31:15)
5 verbose stack     at final (/usr/local/lib/node_modules/npm/node_modules/read-package-json/read-json.js:349:5)
5 verbose stack     at then (/usr/local/lib/node_modules/npm/node_modules/read-package-json/read-json.js:124:5)
5 verbose stack     at ReadFileContext.<anonymous> (/usr/local/lib/node_modules/npm/node_modules/read-package-json/read-json.js:295:20)
5 verbose stack     at ReadFileContext.callback (/usr/local/lib/node_modules/npm/node_modules/graceful-fs/graceful-fs.js:78:16)
5 verbose stack     at FSReqWrap.readFileAfterOpen [as oncomplete] (fs.js:365:13)
6 verbose cwd /Users/Dogukan
7 error Darwin 16.4.0
8 error argv "/usr/local/bin/node" "/usr/local/bin/npm" "start"
9 error node v6.9.5
10 error npm  v3.10.10
11 error Invalid version: "cd"
12 error If you need help, you may report this error at:
12 error     <https://github.com/npm/npm/issues>
13 verbose exit [ 1, true ]
```

How can I resolve this problem?

**Comment by watilde** [comment]
From the official docs:
+ https://docs.npmjs.com/files/package.json#version

It said `"Version must be parseable by node-semver"`, so you need to update the version in your package.json to be parseable by node-semver. To know more about the semver, check this page please: http://semver.org/. Thanks!

**Comment by johnvndnbrk** [comment]
It said what?  

I'm using Windows OS but received the same error.  I fixed it by deleting my npm-cache packages and local node-modules folder.  This may not have been necessary.  Following this I opened up my cmd prompt with elevated privilages and ran "nmp install" and "npm start".  I'm not sure if this will help with your Mac OS but thought I would pass along what worked for me since our error messages where so similar.  Good luck!
